src/warnet/cli/graph.py

```diff
@@ -11,6 +14,68 @@ def graph():
     """Graph commands"""
 
 
+def convert_unsupported_attributes(graph):
+    # Sometimes networkx complains about invalid types when writing the graph
+    # (it just generated itself!). Try to convert them here just in case.
+    for _, node_data in graph.nodes(data=True):
+        for key, value in node_data.items():
+            if isinstance(value, set):
+                node_data[key] = list(value)
+            elif isinstance(value, (int, float, str)):
+                continue
+            else:
+                node_data[key] = str(value)
+
+    for _, _, edge_data in graph.edges(data=True):
+        for key, value in edge_data.items():
+            if isinstance(value, set):
+                edge_data[key] = list(value)
+            elif isinstance(value, (int, float, str)):
+                continue
+            else:
+                edge_data[key] = str(value)
+
+
+def list_graph_generators() -> Dict:
+    modules = [
+        nx.generators.atlas,
+        nx.generators.classic,
+        nx.generators.cographs,
+        nx.generators.community,
+        nx.generators.degree_seq,
+        nx.generators.directed,
+        nx.generators.duplication,
+        nx.generators.ego,
+        nx.generators.expanders,
+        nx.generators.geometric,
+        nx.generators.internet_as_graphs,
+        nx.generators.intersection,
+        nx.generators.interval_graph,
+        nx.generators.joint_degree_seq,
+        nx.generators.lattice,
+        nx.generators.line,
+        nx.generators.mycielski,
+        nx.generators.nonisomorphic_trees,
+        nx.generators.random_clustered,
+        nx.generators.random_graphs,
+        nx.generators.small,
+        nx.generators.social,
+        nx.generators.spectral_graph_forge,
+        nx.generators.stochastic,
+        nx.generators.sudoku,
+        nx.generators.trees,
+        nx.generators.triads,
+    ]
+
+    generators = {}
+    for module in modules:
+        for attr_name in dir(module):
+            if not attr_name.startswith('_') and 'graph' in attr_name:
+                generators[attr_name] = getattr(module, attr_name)
```

Comment:
I don't think you need getattr here, just the function name? bc if this is just for user facing cli, we dont really need to see python junk:

```
    'graph_atlas': <function graph_atlas at 0x104b17f60>,
    'graph_atlas_g': <function graph_atlas_g at 0x104b30040>,
    'barbell_graph': <function barbell_graph at 0x104b305e0>,
    'circulant_graph': <function circulant_graph at 0x104b309a0>,
(etc)
```